app/assets/stylesheets/style.scss

```diff
@@ -1,3 +1,31 @@
+body {
+  background-image: url("http://i.imgur.com/y6o43z3.jpg");
+  background-repeat: no-repeat;
```

Comment:
Properties should be ordered background-image, background-position, background-repeat